tests/templates/access.py

```diff
@@ -53,9 +53,10 @@ def closed_callback():
             logger.debug(f"AccessDialog Close() response {response}")
             cb_success(response.response, response.results)
 
-        def reply_callback():
+        def reply_callback(request):
             response = Response(self.response, {})
             logger.debug(f"AccessDialog with response {response}")
+            request.unexport()
```

Comment:
The ImplRequest already unexports the object when the request is being closed, so I'm not entirely sure why this is needed?

Reply:
But it is not closed. The screenshot portal will use the same request handle for the access and screenshot portal call.  (Actually I just notice without hooking up the access impl request to the app request).

Anyway  if we do not unexport here, the screenshot portal will not be able to add a mock for the request.
` xdg-desktop-portal-WARNING **: 13:41:59.942: A backend call failed: object /org/freedesktop/portal/desktop/request/1_7/request3 already exists`

I checked xdg-desktop-portal-gtk and it works the same. It unexports the request when sending the reponse.

Reply:
Right, I think we have a more fundamental problem here and don't close requests in a lot of cases where we should (i.e. whenever we finished handling the request).

Reply:
To avoid that problem in the future: https://github.com/flatpak/xdg-desktop-portal/pull/1607